packages/peregrine/lib/talons/Logo/useLogo.js

```diff
@@ -0,0 +1,42 @@
+import { useEffect } from 'react';
+import { useQuery } from '@apollo/react-hooks';
+
+/**
+ *
+ * @param {*} props.query the footer data query
```

Comment:
probably the comment needs to be updated here as well - probably logo data query was meant

Reply:
Suggestion added.